gradle/libs.versions.toml

```
@@ -19,8 +19,7 @@ androidJunit = "1.1.5"
 espresso = "3.5.1"
 media3 = "1.2.0"
 media = "1.7.0"
-guava = "31.1-android"
-navigation = "2.7.4"
+navigation = "2.7.6"
 paging = "3.2.1"
 okhttp = "4.10.0"
```

```suggestion
okhttp = "4.12.0"
```

https://square.github.io/okhttp/changelogs/changelog_4x/